continuousflex/__init__.py

```diff
@@ -124,17 +124,14 @@ def defineBinaries(cls, env):
         if os.path.exists(env.getEmFolder() + '/genesis.tgz'):
             os.system('rm ' + env.getEmFolder() + '/genesis.tgz')
 
-        target_branch = "nmmd"
-        env.addPackage('genesis', version='1.4.0', deps=[lapack],
-                       url='https://github.com/mms29/nmmd/archive/%s.tar.gz' %target_branch,
-                       tar='genesis.tgz',
+        target_branch = "nmmd_image_merge"
+        env.addPackage('genesis', version='1.7.1', deps=[lapack],
                        createBuildDir=True,
                        buildDir='genesis',
-                       commands=[('tar -xf ../genesis.tgz -C .;'
-                                  'mv nmmd-%s/* .;'
-                                  'rm -r nmmd-%s;'
+                       commands=[('git clone -b %s https://github.com/mms29/nmmd.git ; '
```

Comment:
We better upload this version of Genesis on [continuousflex-org](https://github.com/continuousflex-org), we have access to it.
Other than that, you know better your codes :-)

Reply:
True, however, I think I am not a contributor, I can not upload